server/src/__tests__/routes/token.test.ts

```diff
@@ -129,18 +128,19 @@ describe('createTokenRoute()', () => {
     });
 
     describe('Verify Twilio Email', () => {
-      it('returns 401 when About Page is enabled, and email is NOT @twilio.com', async () => {
+      it('returns 401 when EMAIL_VERIFICATION_REGEX is provided, and email is NOT @twilio.com', async () => {
         tokenRoute = createTokenRoute({
           ...mockServerConfig,
-          ENABLE_ABOUT_PAGE: 'true',
+          // eslint-disable-next-line no-useless-escape
```

Comment:
What does this rule do for this line?

Reply:
Actually dont need this anymore, removed.